### ISSUE TITLE
Segmentation for lidar

### ISSUE THREAD

**Issue by xslittlegrass** [carla-simulator/carla#516]
There is a "semantic segmentation" camera that provides the segmentation information for camera images. Is there a similar one for lidar, i.e., how can I get class tags for each lidar point?
Thanks!

**Comment by nsubiron** [comment]
Hi @xslittlegrass,

We don't have a Lidar with semantic segmentation tags yet. There has been some attempts on doing that but there are some limitations on how the current lidar implementation works. The main problem is that the ray-casts done by the lidar do not hit the objects but their colliders, and those colliders aren't tagged. See #423 or #455.

**Comment by vishal-prgmr** [comment]
@nsubiron , the code used for simulating the LIDAR in AirSim (developed by MicroSoft) says that the LIDAR implementation is based on the implementation from CARLA. 

I have managed to label the entire point cloud which i obtain in AirSim. Since the implementations are similar should it not be possible to do the same in CARLA. 

P.S. I still havent started exploring CARLA. Will get back to this thread once I have tried something in CARLA.
